pkg/apis/triggers/v1alpha1/interceptor_types.go

```diff
@@ -46,7 +47,26 @@ type InterceptorResponse struct {
 	// Continue indicates if the EventListener should continue processing the Trigger or not
 	Continue bool `json:"continue,omitempty"`
 	// Status is an Error status containing details on any interceptor processing errors
-	Status *status.Status `json:"status,omitempty"`
+	Status Status `json:"status"`
+}
+
+type Status struct {
+	// The status code, which should be an enum value of [google.rpc.Code][google.rpc.Code].
+	Code int32 `json:"code,omitempty"`
```

Comment:
Should we do this instead? This would make it so we don't need to do the codes.Code(code) / int32() casting below.

```suggestion
	Code codes.Code `json:"code,omitempty"`
```

Reply:
done!